src/Editor.js

```diff
@@ -114,6 +114,7 @@ const OptionsErrorMessage = ({ children, style, ...props }) => {
 
 const Editor = () => {
   const canvas = useRef(null);
+  const [editableListItemId, setEditableListItemId] = useState("");
```

Comment:
I reckon we should give this a more description name. It's hard to guess which/what editable list this is talking about or when/why is gets set. Maybe something like `currentlyEditingNameOfLayerId`? It's a of a mouthful, but it's good to avoid cryptic names when the declaration and usage of a variable aren't colocated.

Also, `null` is probably a better default/empty value than `""`.

Reply:
Just so you know, at some point we'll want to abstract the panels out into their own components so that state management like this (state that is only relevant to a small part of the app) can be encapsulated within those components. I haven't pulled the trigger on that yet because premature abstraction is often a bad move (you'll typically make the wrong abstractions if you do it too early because you haven't yet uncovered all the ins-and-outs), but we're probably getting close to the point that it'd make sense.